### PR TITLE
complete

### PR DESCRIPTION
**Pull request by Drewbrad4**

### CODE REVIEW

**Comment by Drewbrad4** [comment]
Booo I'm just realizing that I didn't include a user story to delete a task. But I worked all weekend so I'm too lazy to fix it. So I know that I probably should have done that haha.

File: code-challenges.js [hunk 1]
```diff
@@ -10,9 +10,18 @@ var people = [
...
+  return array.map(object => {
+    let splitName = object.name.split(" ")
+    // uppercase the first character and return to a single string with added sentence. 
+    return splitName.map(name => {
+      return name.charAt(0).toUpperCase() + name.substr(1)
+    }).join(" ").concat(" is a ", object.occupation, ".")
+  }).join(" ")
+}
+
+console.log(peopleInfo(people), "\n");
```
Comment (sjproctor) [inline code-challenges.js:25]: Looks great!

File: code-challenges.js [hunk 2]
```diff
@@ -21,12 +30,30 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
 var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+const numberFilterRemainder = (arr) => {
+  let filteredArr = arr.filter(value => typeof value === "number")
+  return filteredArr.map(num => num % 3)
+}
```
Comment (sjproctor) [inline code-challenges.js:37]: Very clean.

File: code-challenges.js [hunk 3]
```diff
@@ -21,12 +30,30 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
...
+  let joinedArr = []
+  // For each arr passed in, we add in every item to the previous items in joinedArr
+  arr.forEach(array => {
+      joinedArr = [...joinedArr, ...array]
+  }); 
+  // return all items that only have a single index value
+  return joinedArr.filter((item, index) => joinedArr.indexOf(item) === index)
+}
+
+console.log(noDupes(testingArray3, testingArray4))
```
Comment (sjproctor) [inline code-challenges.js:59]: NIce work.

File: interview-questions.md [hunk 4]
```diff
@@ -7,44 +7,55 @@ Then, research the question to expand on your answer. Even if you feel you have
...
+  1. As a user I can give my to do list a name.
+  2. As a user I can add a task to my to do list.
+  3. As a user I can add a description to my task.
+  4. As a user I can mark a task as complete.
+  5. As a user I can see my completed tasks.
+  6. As a user I can unmark a task as complete.
+  7. As a user I can edit the to do list name.
+  8. As a user I can edit the task name and description.
+  9. As a user I can move a task up or down on the to do list, rearranging the order.
+  10. As a user I can delete a task.
```
Comment (sjproctor) [inline interview-questions.md:61]: Nice job getting all the CRUD actions in there.

**Review by sjproctor — APPROVED** [review]
Really great work. I don't have much to add. Nice work, Drew.

**Pull request merged**